### PR TITLE
initalize with mobile medians

### PR DESCRIPTION
**Pull request by paulirish**
`params.get('key_not_in_params')` returns a null, not undefined, so this wasn't working as we expected it to.

fixed and added a few more comments.

intended behavior: if you load the scorecalc with nothing in the URL, you'll see a 50 perf score and each metric has a metric score of 50. nice for playing around.

### CODE REVIEW

File: script/main.js [hunk 1]
```diff
@@ -305,17 +305,17 @@ function getInitialState() {
...
   for (const id in metricScorings) {
     metricValues[id] = metricScorings[id].median;
   }
 
-  // Load from query string.
+  // Populate metricValues from query string.
   for (const [id, metric] of Object.entries(metrics)) {
     const value = params.get(id) || params.get(metric.auditId);
-    if (value === undefined) continue;
+    if (value === null) continue;
```
Comment (connorjclark) [inline script/main.js]:
```suggestion
    if (!value) continue;
```

?

Comment (paulirish) [reply script/main.js]: but... empty strings and 0s.

`URLSearchParams.get()` always returns null if it's not present.

Comment (connorjclark) [reply script/main.js]: oh right.

well then isn't `params.get(id) || params.get(metric.auditId);` wrong for the same reason? really we'd want `??` but not supported.

Comment (paulirish) [reply script/main.js]: > well then isn't `params.get(id) || params.get(metric.auditId);` wrong for the same reason?

yes true. good catch

Comment (connorjclark) [reply script/main.js]: wait, no, because it isn't 0, it's '0'.

And empty string seems like something we'd want to ignore

Comment (paulirish) [reply script/main.js]: yeah i realized that it's '0', too. 

so.. anyway with this PR as merged... you can have a url like `#FCP=2290&SI=2850&FMP=4000&TTI=&FCI=6500&LCP=2080&TBT=260&CLS=0.12&device=mobile&version=6&version=5`  

see that `TTI=` (empty string)..  

in this case, the TTI value is populated with the default, whereas the rest are not. i don't think this is ideal, but ... i also don't think this situation is likely to happen so i kinda don't care.

tldr: whatevz
